docsrc/content/BioCollections.fsx

```diff
@@ -471,7 +471,7 @@ val digestedRBCS' : Digestion.DigestedPeptide [] =
 </div>
 <br>
 ##Nucleotides
-Let's imagine you have a given gene sequence and want to find out what the according protein might look like.
+Let's imagine you have a given gene sequence and want to find out what the corrisponding protein might look like.
```

Comment:
Seems like a typo slipped through here. Can you fix this one before i merge your PR?